Files/Strings/en-US/Resources.resw

```diff
@@ -2794,4 +2791,10 @@ We use App Center to keep track of app usage, find bugs, and fix crashes. All in
   <data name="NoSearchResultsFound" xml:space="preserve">
     <value>No items found</value>
   </data>
-</root>
+  <data name="OpenLogLocation.Title" xml:space="preserve">
+    <value>Open log location</value>
+  </data>
+  <data name="OpenLogLocation.ToolTipService.ToolTip" xml:space="preserve">
+    <value>Open log location</value>
+  </data>
```

Comment:
```suggestion
  </data>
  <data name="SettingsAboutOpenLogLocationButton.ToolTipService.ToolTip" xml:space="preserve">
    <value>Open log location</value>
  </data>
```

For this: (on my testing version I added the 3 to check it)
![image](https://user-images.githubusercontent.com/11770760/130323833-c5829a14-3580-4e2f-9a83-8761bebf96fa.png)